Formula/lexbor.rb

```diff
@@ -0,0 +1,116 @@
+class Lexbor < Formula
+  desc "Fast embeddable web browser engine written in C with no dependencies"
+  homepage "https://lexbor.com/"
+  url "https://github.com/lexbor/lexbor/archive/refs/tags/v1.0.0.tar.gz"
+  sha256 "e9ba3db87b077cde8ee189eb0851e8b11823c778c6844adcce537357dcca8ae1"
+  license "Apache-2.0"
+
+  depends_on "cmake" => [:build, :test]
+
+  def install
+    mkdir "build" do
+      system "cmake", "..", *std_cmake_args
+      system "make"
+      system "make", "install"
+    end
+    mkdir_p "#{prefix}/tests"
+    cp_r "test", "#{prefix}/tests/test"
+    cp_r "config.cmake", "#{prefix}/tests"
+    cp_r "version", "#{prefix}/tests"
+    (prefix/"tests/CMakeLists.txt").write cmakelists_file_content
```

Comment:
Any test code should be set up in the test block only.

Reply:
Thank you for letting me know. As I know, the test dir is temporary but I need access to source dir to get the test subfolder during the test part that is why I'm moving it to Cellar. Do I miss something?